examples/train/deepspeed-cifar.py

```diff
@@ -0,0 +1,83 @@
+# description: train CIFAR-10 using DeepSpeed and PyTorch
+#
+# In this example, we train a PyTorch model on the CIFAR-10 dataset using distributed training via
+# DeepSpeed (https://github.com/microsoft/DeepSpeed) across a GPU cluster.
+#
+
+# imports
+import git
+
+from pathlib import Path
+from azureml.core import Workspace
+from azureml.core import ScriptRunConfig, Experiment, Environment
+from azureml.core.runconfig import MpiConfiguration
+
+# get workspace
+ws = Workspace.from_config()
+
+# get root of git repo
+prefix = Path(git.Repo(".", search_parent_directories=True).working_tree_dir)
+
+# training script
+script_dir = prefix.joinpath("code", "models", "deepspeed", "cifar10")
```

Comment:
this will need to be updated from models => train

Reply:
Fixed.